seed.js

```diff
@@ -0,0 +1,90 @@
+const { users, categories, thread, comments } = require('./models')
+
+const main = async () => {
+    
+    
```

Comment:
Should we add some destroy functions here?
Ex. await User.destroy(), categories.destroy() etc.

Reply:
Ok implemented for next push.